bin/install_cli

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env bash
```

Comment:
@pavelgj this has been copied and modified from the firebase.tools source - could you let us know if this needs correct attribution to anyone and if so, we can add them to the commit/pr?